samples/Common.c

```diff
@@ -352,7 +352,14 @@ STATUS initializePeerConnection(PSampleConfiguration pSampleConfiguration, PRtcP
             CHK_STATUS(signalingClientGetIceConfigInfo(pSampleConfiguration->signalingClientHandle, i, &pIceConfigInfo));
             for (j = 0; j < pIceConfigInfo->uriCount; j++) {
                 CHECK(uriCount < MAX_ICE_SERVERS_COUNT);
-                STRNCPY(configuration.iceServers[uriCount + 1].urls, pIceConfigInfo->uris[j], MAX_ICE_CONFIG_URI_LEN);
+                /* - change url to "turn:ip:port?transport=udp" to do turn through UDP
+                 * - change url to "turn:ip:port?transport=tcp" will do turn through TCP/TLS currently,
+                 *   but later will be changed to plain tcp
+                 * - change url to "turns:ip:port?transport=tcp" to do turn through TCP/TLS
+                 * - "turns:ip:port?transport=udp" will just do UDP.
+                 * Currently if "transport=" is missing, ICE will try both UDP and TCP/TLS
+                 * By default use UDP since it's fastest. */
```

Comment:
I wouldn't do `%s?transport=udp`, you are going to see a slower success rate. With trickle ICE it should just use UDP TURN if it connects first!

Reply:
this is mainly for the alexa case. Trying to do allocation just once